qiskit/primitives/base_estimator.py

```diff
@@ -197,28 +226,27 @@ def parameters(self) -> tuple[ParameterView, ...]:
         """
         return self._parameters
 
-    @abstractmethod
     def __call__(
         self,
-        circuit_indices: Sequence[int],
-        observable_indices: Sequence[int],
```

Comment:
Can we move these two parameters after parameter_values and add a DeprecationWarning. I would have done this in subclass but I don’t see a way with these new changes.

Reply:
It is good idea to use `deprecate_arguments`. I'll add it.

Reply:
Done in 6f9f469. You can use `*_indices` with DeprecationWarning.